.github/workflows/build_and_test.yml

```diff
@@ -1538,7 +1544,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: DOCGroup/ACE_TAO
-        ref: 61338c7 # Update to 6_5_13 when available
+        ref: 9eae9c11f1bcd07b48c74820d8cb4c67aca9ef23 # Update to 6_5_13 when available
```

Comment:
```suggestion
        ref: ace6tao2
```

Reply:
Or the  ACE+TAO-6_5_12 tag?

Reply:
OSX 11.0 support didn't exist for these.